cnames_active.js

```diff
@@ -489,6 +489,7 @@ var cnames_active = {
   "canvg": "canvg.github.io/canvg",
   "canvo": "canvojs.github.io",
   "cap": "cap-js.github.io/docs",
+  "capjs": "tiagorangel1.github.io/cap",
   "capital": "capitaljs.github.io/capitaljs",
```

Comment:
```suggestion
  "capital": "capitaljs.github.io/capitaljs",
  "capjs": "tiagorangel1.github.io/cap",
```
As the CI error notes, your entry needs to be in alphabetical order